modules/apps/sharing/sharing-service/src/main/java/com/liferay/sharing/service/impl/SharingEntryLocalServiceImpl.java

```diff
@@ -171,7 +173,13 @@ public SharingEntry addSharingEntry(
 		sharingEntry.setUserId(user.getUserId());
 		sharingEntry.setUserName(user.getFullName());
 
-		sharingEntry.setToUserId(toUserId);
+		if (toUserGroupId > 0) {
+			sharingEntry.setToUserGroupId(toUserGroupId);
+		}
+		else {
+			sharingEntry.setToUserId(toUserId);
+		}
+
```

Comment:
Why not simplify it to:
```
sharingEntry.setToUserGroupID(...);
sharingEntry.setToUser(...);
```
It is already validated that both cannot be set, so the conditional is superfluous.

Reply:
you are right, thanks